x-pack/plugins/apm/server/lib/services/get_throughput.ts

```diff
@@ -28,9 +27,11 @@ interface Options {
   transactionType: string;
   start: number;
   end: number;
+  intervalString: string;
+  throughputUnit: 'minute' | 'second';
```

Comment:
Should you use the type defined here? https://github.com/elastic/kibana/pull/107850/files#diff-7cb114c4f9fd3fbe294a452e57f20f5e3517f675d7594a0cd35136bb04472081R19

Reply:
Will do 👍